### PR TITLE
Feature/wunderground

### PR DESCRIPTION
**Pull request by israelst**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -10,3 +10,5 @@ pgcli
 sqlsoup
 odo
 geojson
+postgres==2.1.2
```
Comment (fccoelho) [inline requirements.txt]: Any reason for not using psycopg2?

File: wunderground/clima.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+license: GPL v3
+"""
+import argparse
+import datetime
+
+from wu import capture
+from model_postgres import save
+
+
+codes = ['SBRJ',  # santos dumont
```
Comment (fccoelho) [inline wunderground/clima.py]: @israelst please add stations for Minas Gerais

**Comment by fccoelho** [comment]
@israelst I haven't seen any code to fill-in the "Estacao_wu" table. By the way I believe we should refactor this table to include a foreing key to municipio id. Currently it has a link to the localidades table, but it is hard to pin a station on a localidade, withou doing looking at the coordinates and finding which polygon contains the station.

**Comment by fccoelho** [comment]
@israelst , we need to finish this.

File: requirements.txt (hunk 1 above)
Comment (israelst) [inline requirements.txt]: There isn't: 033b7aa

**Comment by fccoelho** [comment]
@israelst,  we will no longer use mongodb to store these data. They should go into pgsql only.
regarding the foreign keys in the table Clima_wu, I already remove the foreign key to localidade_id. All we have is the one to Municipio. So make sure to lookup the municipio's geocodigo before you insert the  data in the clima_wu table.
We will also need to fill in the stations table.

**Comment by israelst** [comment]
Let's leave the filling of stations table for its own PR, I afraid of the size of this one and the possible future merge conflicts, is it ok?

**Pull request merged**